### PR TITLE
CI Test for #35049

### PR DESCRIPTION
**Pull request by gregmagolan**
Testing CI for #35049. Turns on aio monitoring jobs to make sure they are not broken.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -173,8 +173,10 @@
     "vrsource-tslint-rules": "5.1.1"
   },
   "// 4": "Overwrite graceful-fs to a version that does not rely on the 'natives' package. This fixes gulp for >= 10.13, more information: #28213",
+  "// 5": "Ensure a single version of webdriver-manager so it is hoisted as the integration tests depend on it being found at ../../node_modules/webdriver-manager",
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply package.json]: Yarn complains a bit if there are comments there but can be done:
```
warning Resolution field "// 4" does not end with a valid package name and will be ignored
warning Resolution field "// 5" does not end with a valid package name and will be ignored
```

File: aio/karma.conf.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply aio/karma.conf.js]: 👍

File: integration/cli-hello-world-ivy-i18n/karma.conf.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/cli-hello-world-ivy-i18n/karma.conf.js]: 👍

File: integration/cli-hello-world-lazy-rollup/karma.conf.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/cli-hello-world-lazy-rollup/karma.conf.js]: 👍

File: integration/cli-hello-world-lazy/karma.conf.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/cli-hello-world-lazy/karma.conf.js]: 👍

File: integration/cli-hello-world/karma.conf.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/cli-hello-world/karma.conf.js]: 👍

File: integration/hello_world__systemjs_umd/protractor.config.js [hunk 7]
```diff
@@ -1,12 +1,15 @@
...
 exports.config = {
   specs: [
     './e2e/**/*.e2e-spec.js'
   ],
   capabilities: {
     browserName: 'chrome',
     chromeOptions: {
-      'args': ['--no-sandbox'],
-      'binary': process.env.CHROME_BIN,
+      binary: require('puppeteer').executablePath(),
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/hello_world__systemjs_umd/protractor.config.js]: karma requires CHROME_BIN as there is no other way to specify the bin for karma-chrome-launcher. protractor config allows you to set the binary like this.

File: integration/cli-hello-world-ivy-minimal/karma.conf.js [hunk 8]
```diff
@@ -1,5 +1,6 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply integration/cli-hello-world-ivy-minimal/karma.conf.js]: 👍

File: .circleci/config.yml [hunk 9]
```diff
@@ -133,6 +133,25 @@ commands:
...
+      - run:
+          name: Install shared Chrome libs
+          command: |
+            sudo apt-get update
+            # Install GTK+ graphical user interface (libgtk-3-0), advanced linux sound architecture (libasound2)
+            # and network security service libraries (libnss3) & X11 Screen Saver extension library (libssx1)
+            # which are dependendies of chrome & needed for karma & protractor headless chrome tests.
+            # This is a very small install which takes around 7s in comparing to using the full
+            # circleci/node:x.x.x-browsers image.
+            sudo apt-get -y install libgtk-3-0 libasound2 libnss3 libxss1
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply .circleci/config.yml]: The last holdout is still integration/bazel-schematics which I didn't want to touch yet as it is more complicated but all other tests should be able to go to default-executor now. Good call.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (IgorMinar) [reply package.json]: ok. nvm

**Comment by gregmagolan** [comment]
https://github.com/angular/angular/pull/35049 landed

**Pull request closed without merging**